plugins/scaffolder-backend/package.json

```diff
@@ -79,6 +81,7 @@
     "@types/mock-fs": "^4.13.0",
     "@types/nunjucks": "^3.1.4",
     "@types/supertest": "^2.0.8",
+    "esbuild": "^0.13.14",
```

Comment:
aren't we able to make nunjucks a devDep as well?

Reply:
We use it to parse expressions as well so nop : <